code/__HELPERS/names.dm

```diff
@@ -44,69 +31,18 @@ var/religion_name = null
 	return capitalize(name)
 
 /proc/system_name()
-	return "Nyx"
+	return using_map.starsys_name
 
 /proc/station_name()
-	if(station_name)
-		return station_name
-
-	var/name = ""
-
 	if(config && config.server_name)
-		world.name = "[config.server_name]: [name]"
+		world.name = "[config.server_name]: station_name()"

```

Comment:
I feel like a recursive call is perhaps a bad idea here

Reply:
The syntax is incorrect, there is no recursion there. Or any proc call for that matter. *facepalm*